cloud_formation/lambda/auto-build-test/auto-build-test.py

```diff
@@ -152,51 +160,40 @@ def lambda_to_ec2(event, context):
 
 #Endpoint tests:
 echo 'Performing tests...'
-python3.5 ./bastion.py endpoint.test.boss ssh
-cd /srv/www/django
-python3 manage.py test
-python3 manage.py test -- -c inttest.cfg
+python3.5 ./bastion.py --ssh-key ~/.ssh/auto-build-keypair.pem endpoint.test.boss ssh-cmd "cd /srv/www/django && python3 manage.py test"# python3 manage.py test -- -c inttest.cfg
 
 #ndingest library
-python3 -m pip install pytest
-cd /usr/local/lib/python3/site-packages/ndingest
-export NDINGEST_TEST=1
-pytest -c test_apl.cfg
-#Exit endpoint
-exit
+python3.5 ./bastion.py --ssh-key ~/.ssh/auto-build-keypair.pem endpoint.test.boss ssh-cmd "python3 -m pip install pytest"
+python3.5 ./bastion.py --ssh-key ~/.ssh/auto-build-keypair.pem endpoint.test.boss ssh-cmd "cd /usr/local/lib/python3/site-packages/ndingest && export NDINGEST_TEST=1 && pytest -c test_apl.cfg"
 
 #cachemanage VM
-python3.5 ./bastion.py cachemanager.test.boss
-cd /srv/salt/boss-tools/files/boss-tools.git/cachemgr
-sudo nose2
-sudo nose2 -c inttest.cfg
-exit
+python3.5 ./bastion.py --ssh-key ~/.ssh/auto-build-keypair.pem cachemanager.test.boss ssh-cmd "cd /srv/salt/boss-tools/files/boss-tools.git/cachemgr && sudo nose2 && sudo nose2 -c inttest.cfg"
 
 echo " "
 echo "----------------------Delete Stacks----------------------"
 echo " " 
 
-# yes | python3.5 ./cloudformation.py delete test.boss dynamolambda --internal
+# yes | python3.5 ./cloudformation.py delete test.boss dynamolambda
 # wait
-yes | python3.5 ./cloudformation.py delete test.boss cloudwatch --internal
+yes | python3.5 ./cloudformation.py delete test.boss cloudwatch
 wait
-yes | python3.5 ./cloudformation.py delete test.boss activities --internal
+yes | python3.5 ./cloudformation.py delete test.boss activities
 wait
-yes | python3.5 ./cloudformation.py delete test.boss api --internal
+yes | python3.5 ./cloudformation.py delete test.boss api
 wait
-yes | python3.5 ./cloudformation.py delete test.boss redis --internal
+yes | python3.5 ./cloudformation.py delete test.boss redis
 wait
-yes | python3.5 ./cloudformation.py delete test.boss core --internal
+yes | python3.5 ./cloudformation.py delete test.boss core
 wait
 
 # echo " "
 # echo "----------------------Cleanup environment----------------------"
 # echo " " 
 
 #Delete keypairs from aws
-python3.5 ./manage_keypair.py delete auto-build-keypair --internal
-Shutdown the instance an hour after script executes.
-shutdown -h +3600"""
+# python3.5 ./manage_keypair.py delete auto-build-keypair
```

Comment:
Why comment these lines out?

Reply:
I did this during my last testing session, and seems I forgot to put them back in.